### PR TITLE
Update frontend auth to use cookies

### PR DESCRIPTION
**Pull request by Jeremy-Pichon**
Update frontend auth to use cookies

### CODE REVIEW

File: apps/portal-web/app/(protected)/dashboard/page.tsx [hunk 1]
```diff
@@ -96,6 +96,14 @@ export default function DashboardPage() {
     setDeleteModal(true);
   };
 
+  if (loading) {
+    return <></>;
```
Comment (abidon) [inline apps/portal-web/app/(protected)/dashboard/page.tsx:100]: Skeleton or spinner at least ?

File: apps/portal-web/app/(protected)/organizations/page.tsx [hunk 2]
```diff
@@ -1,47 +1,65 @@
...
-    });
-  }, []);
+  const { data, error, loading } = useQuery<gqlDashboardType>(gqlDashboard);
+
+  if (loading) {
+    return <></>;
+  }
+  if (error) {
+    toast.error('Failed to retrieve dashboard');
+    return <></>;
```
[the remark this answers is not in the document]
Comment (abidon) [reply apps/portal-web/app/(protected)/organizations/page.tsx:41]: Also, I think you meant failed to retrieve "organizations" 😉

File: libs/api-connector/src/lib/http.ts [hunk 3]
```diff
@@ -43,8 +101,8 @@ export class Http {
           error: {
             statusCode: 401,
             code: ErrorCode.userAccessUnauthorized,
-            message: 'Unauthorized, please login again',
-          },
+            message: 'Session expired, please login again'
```
[the remark this answers is not in the document]
Comment (Jeremy-Pichon) [reply libs/api-connector/src/lib/http.ts:104]: api-connector.ts & http.ts will be heavily updated and/or removed during the refactoring. So we can ignore this for now.

**Review by abidon — APPROVED** [review]
Not tested but looks OK to me!

**Pull request merged**